plan/client_test.go

```diff
@@ -1,193 +1,56 @@
 package plan
 
 import (
-	"fmt"
 	"testing"
 
+	assert "github.com/stretchr/testify/require"
```

Comment:
I pulled in testify so we can get assertions and stop all this manually constructed boilerplate everywhere which is slow to write, brittle, and gives horrible error messages. Testify's become an internal convention with other Stripe Go projects, so bringing it in here too should be fine.